deploy/k8s/eks/simple-cluster.yml

```diff
@@ -0,0 +1,34 @@
+# A simple example of ClusterConfig object:
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name:
+  region: eu-west-1
+
+vpc:
+  id: 	"vpc-6cb55a15"  # (optional, must match VPC ID used for each subnet below)
+  subnets:
+    # must provide 'private' and/or 'public' subnets by availibility zone as shown
+    public:
+      eu-west-1a:
+        id: "subnet-d4cf96b2"
+
+      eu-west-1b:
+        id: "subnet-b50028fd"
+
+      eu-west-1c:
+        id: "subnet-7a841e20"
+
+nodeGroups:
```

Comment:
Do you think that there is a need for such a machine? we only run agent + cloudbeat there

Reply:
It's the default. I think it's a good choice - so people won't run into resource exhaustion errors.